homeassistant/components/squeezebox/media_player.py

```diff
@@ -197,8 +199,10 @@ def __init__(
         self._previous_media_position = 0
         self._attr_unique_id = format_mac(player.player_id)
         _manufacturer = None
+        _entry_type = None
         if player.model == "SqueezeLite" or "SqueezePlay" in player.model:
             _manufacturer = "Ralph Irving"
+            _entry_type = DeviceEntryType.SERVICE
```

Comment:
That doesn't look right.  All the squeezelite's will show up as Services, not devices.

Reply:
yup was down as a service in the tests

Reply:
Tweaked now for nonn service

Reply:
👍